spec/models/notification_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Let's write some specs in here!  Even though `Notification` is a tiny model, let's ensure it's properly associated.  Take a look at some of the other `spec/models` files - there will be an example of something like "`it should belong_to X`"